### ISSUE TITLE
Ubuntu 16.04: utf8 missing?

### ISSUE THREAD

**Issue by daniela-waranie** [chef/bento#804]
I run this box: bento/ubuntu-16.04

And tried to use autofs to mount some samba shares and noticed that i can mount it when i ommit the "iocharset=utf8" parameter. But when i use the iocharset paramter i get this error:

```
attempting to mount entry /var/ncdatadir/qnap
>> mount error(79): Can not access a needed shared library
>> Refer to the mount.cifs(8) manual page (e.g. man mount.cifs)
```
What i did before:
```
sudo apt-get update
sudo apt-get install -y autofs cifs-utils linux-image-extra-virtual
reboot
sudo mkdir -p /var/ncdatadir
sudo nano /etc/auto.master
#add: /var/ncdatadir   /etc/auto.mountqnap
sudo nano /etc/auto.mountqnap
#add: qnap -fstype=cifs,rw,user=myusername,password=mypassword,file_mode=0770,dir_mode=0770,nounix,uid=33,gid=33,iocharset=utf8 ://192.168.8.4/myshare
service autofs stop
#ensure via ps aux if there is another automount process, kill it 
#to bring debugging info to the foreground:
sudo automount -v -f
#launch a 2nd terminal.
cd /var/ncdatadir/qnap
```

First issue at a vanilla box was that /sbin/mount.cifs was missing. That is the reason why i installed cifs-utils. Then the above error appeared. Then followed these tips here:
https://askubuntu.com/questions/519796/unable-to-mount-cifs-with-iocharset-utf8-in-trusty

```
uname -r
4.8.0-51-generic
```

I can confirm that ls /lib/modules/$(uname -r)/kernel/fs/nls/nls_utf8.ko is missing:
`ls: cannot access '/lib/modules/4.8.0-51-generic/kernel/fs/nls/nls_utf8.ko': No such file or directory
`Means the complete directory "nls" (natual language support) is missing.

 dmesg shows this error:
[   12.549080] random: crng init done
[  247.824290] FS-Cache: Loaded
[  247.838457] FS-Cache: Netfs 'cifs' registered for caching
[  247.839087] Key type cifs.spnego registered
[  247.839089] Key type cifs.idmap registered
[  247.840226] CIFS VFS: CIFS mount error: iocharset utf8 not found

apt-get install -y linux-image-extra-virtual (version 4.4.0.77.83) should install that, but it seams to be still missing.

I am absolute confused that nower days UTF8 is still not included in a minimal install. Kernel modules, and such stuff is really hard to me...

How to fix that? Or is there an ubuntu 16.04 full install available for download with vagrant?

**Comment by rmoriz** [comment]
This is an ubuntu issue as bento just builds a virtual image out of the default minimal installation.

**Comment by daniela-waranie** [comment]
Thank you very,very much. I did not expect that the latest release is somehow incomplete.

I installed linux-image-extra-4.4.0-21-generic and rebooted. But it seams that the kernel do not pick hat module. Still the same error as above. uname -r says: 4.8.0-51-generic

I'm not sure, but i think i need to boot into linux-image-4.4.0-21-generic - but how to tell grub to load that old kernel at next boot without GUI? (i am using virtualbox in headless mode on a server)
